WPMedia/fusion-news-theme-blocks
# Removed 'pos' targeting value from ads-block

## Description
Removed 'pos' targeting value from ads-block

## Jira Ticket
- [PEN-1223](https://arcpublishing.atlassian.net/browse/PEN-1223)

## Acceptance Criteria
`pos` targeting param should be removed but `position` should still be present

## Test Steps
1. In `Fusion-News-Theme` repo, checkout `master` & `git pull`
2. After checking out this feature branch in `fusion-news-theme-blocks`, run `rm -rf ./node_modules && npx lerna clean && npm i && npx lerna bootstrap && cd blocks/header-nav-chain-block && npm i && cd ../..`
3. Run `npx fusion start -f -l @wpmedia/header-nav-chain-block` in `Fusion-News-Theme`
4. Open homepage in PB editor and make sure two instances of `Google Ad` feature with a type of `leaderboard_large` are added to the page. 
5. Add a couple other instances of other ad types (doesn't really matter which types) - then stage & publish.
6. Open http://localhost/pf/homepage/?_website=the-sun in a new tab
7. Open Chrome DevTools `Network` tab and filter by `dfp`
8. Find the requests for the `leaderboard_large` ad calls and look for the `prev_scp` param (under query string params)
9. There should no longer be a `pos` param but there **should** be `position` params. The first of the `leaderboard_large` instances should have a `position` of `1` and the second instance should have a `position` of `2`.
10. Check the other ad requests to ensure they have a `position` param

## Effect Of Changes
### Before
Ad calls included `pos` targeting value in DFP request

### After
Ads no longer include the `pos` targeting value in DFP requests

## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
